### DISCUSSION TITLE
Requested transcription of arabic audio with detect_language got back english transcription

### DISCUSSION THREAD

**Discussion by tookam** [deepgram/community#543] [General help] [answered]
### Which Deepgram product are you using?

Deepgram Console

### Details

transaction_key:"deprecated"
request_id:"78eba4e8-4ae6-43c1-a55b-fb3746774392"
sha256:"7722e69284e4fdc5e0ebf3e1d64c1aec85bb4d6a7f3267b83106aaee9a931319"
created:"2024-01-19T11:10:22.976Z"
duration:300.024
channels:1
0:"2a000f0d-afa1-4f20-8481-94f8bc30b0b2"
name:"large-whisper"
version:"2022-12-08.0"
arch:"whisper"

### If you are making a request to the Deepgram API and have a request ID, please paste it below:

78eba4e8-4ae6-43c1-a55b-fb3746774392

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @tookam, our [language detection](https://developers.deepgram.com/docs/language-detection) feature currently supports 16 languages, which does not include Arabic. In order to use our managed Whisper model for Arabic transcription, you'll need to explicitly pass the parameter `language=ar`, and then you will receive a transcript back in Arabic.

**Reply by tookam** [reply]
@jkroll-deepgram thanks for feedback .. I believe I tried that and got back non-arabic transcript (I think english) .. but will try again and report back
